dotnet/src/SemanticKernel.Core/Planning/Plan.cs

```diff
@@ -359,6 +359,7 @@ public FunctionView Describe()
                 this.UpdateContextWithOutputs(context);
 
                 result = new FunctionResult(this.Name, this.PluginName, context, context.Result);
+                this.UpdateFunctionResultWithOutputs(result);
```

Comment:
This added functionality could use some unit tests.

Reply:
Thanks for the callout. This now has me reconsidering a few things. Marking as draft for now.